### DISCUSSION TITLE
watchExclude

### DISCUSSION THREAD

**Discussion by Guid75** [vitest-dev/vitest#4091] [General]
Hi,

`watchExclude` is fine but I'm wondering why we have to put in there all files we don't want to watch, would it be easier to have a "`watchInclude`" property too?
In my case, it would be `["src/**", "test/**"]`.

Is this something that would be accepted? Or has it been rejected before?

**Comment by sheremet-va** [comment]
You can use `server.watch.include` for that I think.

**Reply by Guid75** [reply]
"include" does not exist, vitest uses vite that uses the chokidar API
